README.md

```diff
@@ -71,6 +71,22 @@ git clone git@github.com:Luffytaro22/Blog-App.git
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Install
+To install all the ruby gems, use the following command:
+```
+bundle install
+npm install
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+To run the RSpec unit tests, use the following command:
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 <!-- AUTHORS -->
 
 ## 👥 Authors <a id="authors"></a>
```

Comment:
- [ ] Please kindly fix the `linkedln and issues` broken links from the README file, implementing this will make your repo to be more engaging and informative. 👍